core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -827,6 +982,7 @@ export class LoggedInUser extends BaseUser {
     );
     const newTabPage = await newTarget.page();
     await newTabPage?.waitForNetworkIdle();
+
     expect(newTabPage?.url()).toContain(
       'https://accounts.google.com/lifecycle/steps/signup/name'
```

Comment:
Is it possible to store the link as a constant in test-constant.ts?

Reply:
Done